app/uk/gov/hmrc/nrs/retrieval/config/WSHttp.scala

```diff
@@ -53,9 +55,7 @@ trait WSHttpT extends HttpGet with WSGet
 class WSHttp @Inject()(val environment: Environment, val runModeConfig: Configuration, val appNameConfig: Configuration, val wsClient: WSClient)
                       (implicit val actorSystem: ActorSystem) extends WSHttpT {
 
-  override val hooks = NoneRequired
-
-  // todo : unclear what we need to do with the config here
-  override protected def configuration: Option[Config] = None
+  override val hooks: Seq[HttpHook] = NoneRequired
 
+  override protected def configuration: Config = runModeConfig.underlying
```

Comment:
`configuration` is no longer optional - `runModeConfig.underlying` seems harmless (we had `None` before).